functests/performance/performance.go

```diff
@@ -22,6 +23,15 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 )
 
+var profileName string
+
+func init() {
+	profileName = os.Getenv("PERF_TEST_PROFILE")
```

Comment:
So we would still have the profilename at 2 places, in the manifest, and in the env var for tests.
Can we put the env var into the Makefile, and use it in the manifest as well? Should work because we pipe manifests through envsubst before applying them.

Reply:
So if I understood correctly, you mean getting rid completely of the hardcoded one ("ci") and always relying on the env variable one right?

Reply:
Also, does the same apply to `RoleWorkerRT` ? Because I am seeing that in many places and it ends up being a lot less readable if we set it from an env variable.

Reply:
Another point: the envsubst is done at `ci-cluster/performance` level, not on base : https://github.com/openshift-kni/performance-addon-operators/blob/ae7d00b169ae271d25aff28071117e5dc108d4bc/hack/deploy.sh#L30

I don't see an easy way to pipe the envsubst here because `base` is read from kustomize. Any ideas? Can the duplication of `ci` be handed later as it was already in place? 
I am only adding a way to override its default value here.

Reply:
forget what I wrote. I missed the point that you want to run tests against a cluster which was not prepared by this repo.